src/shared/ui/SearchForm/SearchForm.test.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { SearchForm } from "./SearchForm";
+import { keyboard } from "@testing-library/user-event/dist/keyboard";
+
+describe("SearchForm Component Functionality", () => {
+  test("Should Render Initial Value", () => {
+    const initialValue = "Test";
+    render(<SearchForm initialValue={initialValue} />);
+    const inputElement = screen.getByRole("textbox");
```

Comment:
Let's use a special tag attribute `data-testid` instead 
e.g `data-testid="search-input"`
`getByTestId("search-input")`

https://testing-library.com/docs/queries/bytestid/

Reply:
done

Reply:
you still use `getByRole` instead of `getByTestId`

Reply:
sorry I had left it in one place

done